object/saml_idp.go

```diff
@@ -179,8 +179,8 @@ type IdpSSODescriptor struct {
 }
 
 type NameIDFormat struct {
-	XMLName xml.Name
-	Value   string `xml:",innerxml"`
+	//XMLName xml.Name
```

Comment:
What's this PR? You commented two fields and add two prints? Why this PR will work?